### ISSUE TITLE
menu: increase exander left margin

### ISSUE THREAD

**Issue by sedyh** [McShelby/hugo-theme-relearn#943]
Hello, some of my friends and I noticed that the default margins for navigation icons are almost absent, they are quite difficult to access in the desktop version of the page.

1. Is it possible to increase the margin a little?

2. It would also be nice to have a same fixed size for the open and closed sections.

![breadcrumps](https://github.com/user-attachments/assets/3b3dc195-a611-4256-a290-f7195a49855b)

**Comment by McShelby** [comment]
I agree that this would be preferable, but due to time constraints it is unclear when or even whether I will address this in the future.

**Comment by sedyh** [comment]
I guess I could try to help with this. It doesn't seem very hard to implement.

**Comment by McShelby** [comment]
I appreciate your help!

Things to keep in mind: 7.1.0 introduced submenus for the shortcut / Hugo menus. This is not exposed in the exampleSite. Those shortcut menus are shown in a slightly compressed form, which has influence on the placement of the expander. So, this needs to be tested, too.

**Comment by McShelby** [comment]
I can not see, how to gain extra space for the click area without touching the size and position of the current output. Later is not preferred, so I'll close this for now.

I did move the chevron-right expande a tad to the right but this will not increase the clickable area.
